src/State/OrganisationTreeProvider.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\State;
+
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\State\ProviderInterface;
+use App\Exception\ProviderException;
+use App\Repository\OrganisationDataRepository;
+
+readonly class OrganisationTreeProvider implements ProviderInterface
```

Comment:
It's not a _tree_ you provider, but a _path_ in a tree.

Reply:
Renamed to _path_.